src/components/auth/ImagePicker/index.tsx

```diff
@@ -1,21 +1,35 @@
 import camera from '@/assets/images/camera.png';
 import defaultProfileImg from '@/assets/images/defaultProfile.png';
-import { useState } from 'react';
+import { ReactEventHandler, useState } from 'react';
 import { useFormContext } from 'react-hook-form';
 import * as Styled from './ImagePicker.styled';
+import { useImageUpload } from '@/hooks/@common/useImageUpload';
 
 export interface IImagePickerProps {
   name: string;
+  usage: 'signUp' | 'myPage'; // usage: 사용처(회원가입, 마이페이지)
+  imageUrl: string;
 }
 
-const ImagePicker = ({ name }: IImagePickerProps) => {
+const ImagePicker = ({ name, usage, imageUrl }: IImagePickerProps) => {
   const { register, setValue } = useFormContext();
 
-  const [pickedImage, setPickedImage] = useState<string | null>(null);
+  const [pickedImage, setPickedImage] = useState<string | null>(imageUrl);
+
+  const handleImgError: ReactEventHandler<HTMLImageElement> = (e) => {
```

Comment:
```suggestion
  const handleImgError: ReactEventHandler<HTMLImageElement> = () => {
```
e는 사용하지 않아서 제거해도 좋을 것 같습니다.

Reply:
수정했습니다!